### DISCUSSION TITLE
Timeseries doesn't load in ETL

### DISCUSSION THREAD

**Discussion by SbiCA** [ravendb/ravendb#17904] [Q&A]
I'm struggling to get a timeseries loaded. 

I followed the documentation https://ravendb.net/docs/article-page/6.0/csharp/server/ongoing-tasks/etl/raven#time-series
``` javascript
loadToEventDocuments(this);


function loadCountersOfEventDocumentsBehavior(documentId, counterName) {
   return true;
}

function loadTimeSeriesOfEventDocumentsBehavior(docId, timeSeriesName) {
   // only normal ts
   if(timeSeriesName =="AppOpen"){
       return true;
   }
   return false
}
```

my collection is called `EventDocuments` and I have an ETL setup just for this one collection with the following script for what ever reason the time series `AppOpen` doesn't appear on the target. 

This I tried to make the change

1. reset ETL

2. recreate ETL

3. delete the existing document on the target:
   - change source doc
   - reset ETL to load doc

Also to give the complete context
Source server is a raven cloud instance running 5.4.112
Target server is self hosted instance running 5.4.112 

I'm pretty much clueless

**Comment by aviv86** [comment]
Hi 

This is a bug.. apparently we are passing the lower time series name as the `timeSeriesName` argument.

I created a ticket to fix that, you can track it here:
https://issues.hibernatingrhinos.com/issue/RavenDB-21866/ETL-TimeSeries-load-behaviour-function-passes-lower-timeseries-name-as-argument

As a workaround, you can use:
```
function loadTimeSeriesOfEventDocumentsBehavior(docId, timeSeriesName) {
   // only normal ts
   if (timeSeriesName =="appopen"){
       return true;
   }
   return false
}
```

Regards

**Reply by SbiCA** [reply]
Thanks for looking into it @aviv86  

I can confirm after switching the script it worked for me but for what ever reason the suggested lower casing didn't to the trick I switched to the following. 

``` javascript
function loadTimeSeriesOfEventDocumentsBehavior(documentId, timeSeriesName) {
   // only normal ts
   if(!timeSeriesName.startsWith("inc")){
       return true;
   }
   return false
}

```

**Reply by SbiCA** [reply]
I'm wondering if this shouldn't become the default anyway since incremental time series are not supported yet 

see https://github.com/ravendb/ravendb/discussions/15193
